spec/interactors/projects/destroy_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe Projects::Destroy do
+  describe ".call" do
+    let(:interactor) { described_class.new(project: project, users: users) }
+    let!(:project) { create(:project) }
```

Comment:
```suggestion
    let!(:project) { create :project }
```
Можно не указывать скобки